pandas/tests/tseries/offsets/test_offsets_properties.py

```diff
@@ -85,8 +85,6 @@
 # Offset-specific behaviour tests
 
 
-# Based on CI runs: Always passes on OSX, fails on Linux, sometimes on Windows
-@pytest.mark.xfail(strict=False, reason="inconsistent between OSs, Pythons")
```

Comment:
It says here that on linux it always fails (at least this is how I understood it), but it passed on my machine.

Reply:
Ok so I think I understood why this failing:

```hypothesis.extra.dateutil.timezones``` is using ```dateutil.UTC``` which is avaible only from ```dateutils>=2.7.0``` (https://dateutil.readthedocs.io/en/stable/tz.html#dateutil.tz.dateutil.tz.UTC)

but macOS have a pinned ```python-dateutils``` to ```2.6.1``` which then leads to errors.

Reply:
> is bumping python-dateutils to 2.7.0 is an option/reasonable thing to do?

2.7.0 is 2 days shy of 2 years old, so I think we're OK bumping

Reply:
@MomIsBestFriend this is definitely an improvement, but we can also just bump the dateutil requirement

Reply:
> @MomIsBestFriend this is definitely an improvement, but we can also just bump the dateutil requirement

Sure! will do that in a separate PR